### PR TITLE
Better library updates

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Restricted the scope of my changes a little more, and took Yingjie's advice about the SQL query. @stkem @yingjieMiao

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/RecommendationsCommander.scala [hunk 1]
```diff
@@ -138,12 +138,21 @@ class RecommendationsCommander @Inject() (
...
+    seqOfSeqs.flatMap(_.take(maxPerSeq))
+  }
+  def maybeUpdatesFromFollowedLibraries(userId: Id[User], maxUpdates: Int = 20, maxUpdatesPerLibrary: Int = 5): Future[Option[FullLibUpdatesRecoInfo]] = {
     val keepsOpt: Option[Seq[Keep]] = db.readWrite { implicit session =>
       val lastSeen = userValueRepo.getValue(userId, UserValues.libraryUpdatesLastSeen)
       if (lastSeen.isBefore(currentDateTime.minusHours(12))) {
         userValueRepo.setValue(userId, UserValueName.UPDATED_LIBRARIES_LAST_SEEN, currentDateTime)
-        Some(keepRepo.getRecentKeepsFromFollowedLibraries(userId, 20))
+
+        val recentlyUpdatedKeeps = keepRepo.getRecentKeepsFromFollowedLibraries(userId, 5*maxUpdates)
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RecommendationsCommander.scala:150]: there should be space around `*`. If you compile it, `scalariform` will complain and auto correct it. Be sure to compile before commit. Yes, it's easy to forget.

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RecommendationsCommander.scala:150]: Ah, that's intended behavior. I was wondering why I kept getting spaces.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/RecommendationsCommander.scala:150]: haha, sometimes it's annoying.

**Comment by stkem** [comment]
:+1:

**Comment by stkem** [comment]
I actually kinda liked the extra comments :-)

**Comment by ryanpbrewster** [comment]
Maybe as a separate commit.

**Pull request merged**